### ISSUE TITLE
Output window support color

### ISSUE THREAD

**Issue by g199209** [jimasp/behave-vsc#30]
When we run baheve directly from the terminal, it has colored output, can we support it in output window?

**Comment by jimasp** [comment]
TLDR: won't implement.

MS didn't support coloured output when the extension was created. i think they do now, but i don't think i will be implementing colour anyway, because to work better with the vscode test UI, the output should actually be in the test output window NOT in a separate extension output window,  and the test output window doesn't support colour atm. (I have just not got around to moving the output to the test output window yet since they supported multiroot as it was a recent change.)

Additionally I'm not sure if it would be possible to do this regardless, currently the text comes from the nodejs "spawn" package's output stream, i.e. afaik it just gives you a text stream with some ansi escape sequences, which doesn't include the colour (unless its an option in spawn, I've not checked in detail how the stream works). So you'd have to re-implement colours based on the incoming text, which would be messy.

Btw, the reason that the extension currently uses the output window rather than the test output window is because MS did not initially support multiroot workspace test output windows. they do now, but i haven't got around to implementing the change.
